src/utils.c

```diff
@@ -746,3 +747,25 @@ enum ibpi_pattern get_ibpi_for_value(const int value, const struct ibpi_value ib
 	}
 	return tmp->ibpi;
 }
+
+void inline static no_op() {};
+void inline static prefix_json() { printf("["); };
+void inline static separator_json() { printf(" , "); };
+void inline static suffix_json() { printf("]\n"); };
```

Comment:
We make them static but later we set those functions as callbacks.  That confuses me.
Can we define them in header file as not static?

Reply:
The idea was that the implementation of each callback would be private, to prevent them from being called directly in other parts of the code.  We can increase the visibility of the functions as requested.